src/components/TextBox/index.tsx

```diff
@@ -471,6 +480,14 @@ const TextBox = forwardRef<
     const hasRightElement =
       valid || required || isPassToggleable || random || unit
 
+    const getRightElementPadding = () => {
+      if (required && hasRightElement) {
+        return 22
```

Comment:
What is this number ? 🤔

Reply:
It allows to have the required icon centered:

<img width="98" alt="Screenshot 2021-09-16 at 15 15 00" src="https://user-images.githubusercontent.com/43268759/133621416-13d9572f-e4f6-45ab-a45d-6650c5a62a32.png">

Reply:
Possible to put it in a const describing it pls ? :)